controller/src/main/java/io/pravega/controller/store/stream/PravegaTableScope.java

```diff
@@ -39,7 +39,7 @@
  * Each such scope table is protected against recreation of scope by attaching a unique id to the scope when it is created. 
  */
 public class PravegaTableScope implements Scope {
-    private static final String STREAMS_IN_SCOPE_TABLE_FORMAT = "Table" + SEPARATOR + "streamsInScope" + SEPARATOR + "%s";
+    private static final String STREAMS_IN_SCOPE_TABLE_FORMAT = "Table" + SEPARATOR + "%s" + SEPARATOR + "streamsInScope" + SEPARATOR + "%s";
```

Comment:
I understand that the motivation for changing the naming structure of Table Segments used for metadata is that, currently, the metrics for these segments are not correctly reporting the `scope` tag, right? If this is the case, I have two questions:
1) Have you tested that the method `segmentTags()` in `MetricsTags.java` is correctly reporting the scope as a tag for metadata Table Segments? Maybe we need a test for that, which is related to this comment: https://github.com/pravega/pravega/pull/3594#pullrequestreview-226346168
2) As I understand the underlying problem, all the Table Segments created by users that do not follow the same structure as the one used here, will also have the problem of not correctly reporting the `scope` tag?

Reply:
@RaulGracia I'll let @shiveshr answer you questions, but I want to address this specific point:

> I understand that the motivation for changing the naming structure of Table Segments used for metadata is that, currently, the metrics for these segments are not correctly reporting the scope tag

For me, the main motivation is that the tables of streams in scopes are in the user scopes themselves, and I think they shouldn't because that's really system metadata and should not live in user scopes. Moving the tables to the system scope also affects the metrics names as you are well pointing out.

Reply:
These are internal tables created by controller where it needs a predictable functional computation for the table name from scope and stream.. 
the user created table segments will have scope/table-name.


I am not sure about tags though. I have not tested tags behaviour explicitly.  I will check with you on it and update here

Reply:
My sense is that tags is out of scope of this PR, although clearly it is something we need to look into.

Reply:
I agree that it may be out of scope wrt this change set. But I want to understand how the tags are extracted and used so that we can evaluate whether we are breaking some other expectation vis a vis tags. 

So @RaulGracia could you please summarize how do metrics infrastructure apply scope tags? And what scope tag do we want for internal table segments. 

If the scope tag is extracted from the Fully Qualified segment name, then the scope tag earlier would have been user scope and now would be system scope. 

What should be the expectation there? If its system scope, then we do not need to do anything additional.